test/uk/gov/hmrc/tai/viewModels/income/BbsiClosedCheckYourAnswersViewModelSpec.scala

```diff
@@ -95,6 +95,6 @@ class BbsiClosedCheckYourAnswersViewModelSpec extends PlaySpec
     }
   }
 
-  val dateBeforeCurrentTaxYear = TaxYearResolver.startOfCurrentTaxYear.minusDays(1)
-  val dateWithinCurrentTaxYear = TaxYearResolver.startOfCurrentTaxYear
+  val dateBeforeCurrentTaxYear = TaxYear().endPrev
```

Comment:
Is this the same as:
TaxYearResolver.startOfCurrentTaxYear.minusDays(1)

Reply:
Yes